project/Dependencies.scala

```diff
@@ -99,22 +99,22 @@ object Dependencies {
     object Backend {
       private val version =
         "2.0.4"
-      val logPlay = "io.dataswift" %% "log-play" % version
-      val hatPlay = "io.dataswift" %% "hat-play" % version
-      val dexPlay = "io.dataswift" %% "dex-play" % version
+      val logPlay = "io.dataswift" %% "log-play" % version exclude ("com.typesafe.play", "*")
+      val hatPlay = "io.dataswift" %% "hat-play" % version exclude ("com.typesafe.play", "*")
+      val dexPlay = "io.dataswift" %% "dex-play" % version exclude ("com.typesafe.play", "*")
```

Comment:
this is compiled against play-json 2.8.1, it also pulls in test libraries... we really need to sort our dependencies!

Reply:
How difficult would be to update to play 2.8 this project? It is a scope creep for sure but if we fail to make it compile and run maybe updating is the fastest solution?

Reply:
This fix will not work becuase it seems the versions of play look for different artifacts on the class path, so the HAT version and Library version cannot be different.
The dependencies are an absolute mess
It was not scoped but i think dealing with it is unavoidable
upgrading to 2.8. seems like the proper fix, but it is also pretty risky.
having different versions of the libraries with different play versions is probably the simplest solution but its messy and hard to maintain